ArchiSteamFarm/IPC/Startup.cs

```diff
@@ -130,8 +133,38 @@ internal sealed class Startup {
 
 			// The order of dependency injection matters, pay attention to it
 
+			// Add support for custom reverse proxy endpoints
+			List<string>? knownProxiesText = Configuration.GetSection("Kestrel").GetValue<List<string>?>("KnownProxies");
```

Comment:
`GetValue<>` can't provide us with `HashSet<string>` instead? Have you checked if it can do the `IPAddress` parsing right away?

Reply:
IPAddress nor IPNetwork can't be parsed this way, so we've to parse them manually. Will change to HashSet though in the following commit.